webshop-backend/src/shopping/ShopCurrency.java

```diff
@@ -0,0 +1,5 @@
+package shopping;
+
+public enum ShopCurrency {
+    HUF,EUR,USD,GBP
+}
```

Comment:
so as I see we are planning to support all currencies @deb05121

Reply:
ok